### ISSUE TITLE
/match/v4/matchlists totalgames is not affected by season id

### ISSUE THREAD

**Issue by Colorfulstan** [RiotGames/developer-relations#62]
Example request:
https://euw1.api.riotgames.com/lol/match/v4/matchlists/by-account/eim6EwuDC-OOFh4j67ikRDYt6zZXgq1Ht0SyAe8WlazED_k?champion=126&queue=420&season=11&endIndex=99&beginIndex=0&api_key=**<APIKEY>**

Result:
```
{
    "matches": [],
    "endIndex": 4,
    "startIndex": 0,
    "totalGames": 4
}
```

If you remove the season parameter, it returns the 4 matches (played in seson:9) correctly.

**Comment by MingweiSamuel** [comment]
(From what Andre told me), the season parameter is fundamentally broken

related #84

**Comment by AP-Cohen** [comment]
Wrong, use the Season #13, thats season 9

**Comment by Colorfulstan** [comment]
Not sure what you're telling me there

**Comment by AP-Cohen** [comment]
https://euw1.api.riotgames.com/lol/match/v4/matchlists/by-account/eim6EwuDC-OOFh4j67ikRDYt6zZXgq1Ht0SyAe8WlazED_k?champion=126&queue=420&season=13&endIndex=99&beginIndex=0&api_key=

Change the season to season=13 as I did above

**Comment by Colorfulstan** [comment]
That doesn't concern the issue. I guess it's related to totalGames being fkd up generally anyways.

**Comment by AP-Cohen** [comment]
I agree, that should be something that is fixed. Also wish I wouldn't get a 404 when within a timestamp there are 0 games :/

**Comment by tisbells** [comment]
Closing out old issues. Match-v4 is no longer supported, feel free to re-open or comment if you feel this is in error.
